### PR TITLE
update LPR model input according spec

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

**Comment by eaidova** [comment]
@Wovchena I am not sure that current solution for demo is the best, please take a look and suggest better fix if possible

File: demos/security_barrier_camera_demo/cpp/net_wrappers.hpp [hunk 1]
```diff
@@ -283,13 +283,10 @@ class Lpr {
...
         if (LprInputSeqName != "") {
             InferenceEngine::Blob::Ptr seqBlob = inferRequest.GetBlob(LprInputSeqName);
-            // second input is sequence, which is some relic from the training
-            // it should have the leading 0.0f and rest 1.0f
             InferenceEngine::LockedMemory<void> seqBlobMapped =
                 InferenceEngine::as<InferenceEngine::MemoryBlob>(seqBlob)->wmap();
             float* blob_data = seqBlobMapped.as<float*>();
-            blob_data[0] = 0.0f;
-            std::fill(blob_data + 1, blob_data + seqBlob->getTensorDesc().getDims()[0], 1.0f);
+            std::fill(blob_dat, blob_data + seqBlob->getTensorDesc().getDims()[0], 1.0f);
```
Comment (eaidova) [inline demos/security_barrier_camera_demo/cpp/net_wrappers.hpp:289]:
```suggestion
            std::fill(blob_data, blob_data + seqBlob->getTensorDesc().getDims()[0], 1.0f);
```

**Review by vladimir-dudnik — APPROVED** [review]

**Review by Wovchena — APPROVED** [review]

**Comment by eaidova** [comment]
Jenkins please retry a build

**Comment by eaidova** [comment]
Jenkins please retry a build

**Pull request merged**